ludwig/utils/audio_utils.py

```diff
@@ -64,8 +63,11 @@ def read_audio_from_path(path: str) -> Optional[TorchAudioTuple]:
 
     Useful for reading from a small number of paths. For more intensive reads, use backend.read_binary_files instead.
     """
-    bytes_obj = get_bytes_obj_from_path(path)
-    return read_audio_from_bytes_obj(bytes_obj)
+    try:
+        return torchaudio.backend.sox_io_backend.load(path)
+    except Exception as e:
+        logger.warning(e)
```

Comment:
I wonder if we should just raise an error.

@tgaddair @w4nderlust @connor-mccorm it feels like supporting invalid audio files just bites users later when they start investigating model quality. What do you think?